x/authz/exported/authorizations.go

```diff
@@ -10,14 +10,27 @@ import (
 type Authorization interface {
 	proto.Message
 
-	// MethodName returns the fully-qualified Msg service method name as described in ADR 031.
-	MethodName() string
+	// MsgTypeURL returns the fully-qualified Msg service method name as described in ADR 031.
+	MsgTypeURL() string
```

Comment:
I wonder if we could just do something like this:

```suggestion
	MsgType() sdk.Msg
```

What do you think?

Reply:
So to remove `URL` everywhere? (in all `msg_type_url` names)

Reply:
No. SendAuthorization already knows it's concrete type url for example. Just an idea...

Reply:
We decide to keep the `MsgTypeURL() string` - it's easier to reason. We don't need an instance here. Only a type name for routing.